software/contrib/traffic.md

```diff
@@ -0,0 +1,21 @@
+# Traffic
+
+This script is inspired by [Jasmine & Olive Tree's Traffic](https://jasmineandolivetrees.com/products/traffic).
+Instead of 3 gate inputs, this version only has 2.
```

Comment:
[optional] I think that this script could benefit from a more detailed description, including some details for how it should be used. I tried to create a patch using it as a preset selector for plaits (as mentioned on the Traffic module's page), but it became apparent very quickly that I was missing something fundamental to how it behaves.

Reply:
I think at the time I wrote it I actually misunderstood how the original Traffic worked. I thought each channel acted like a mixer, but it's in fact more of a switch. The dangers of re-implementing a module I don't own and have never seen in person.

I'll need to make a few changes to the implementation to bring it more in line with how "real" Traffic works.

Reply:
And done. I've re-implemented how the script works so it's actually much closer to the _real_ Traffic module now, just with 2 inputs + two additional difference outputs. I've also expanded the readme so it's hopefully clearer how it's actually intended to work.